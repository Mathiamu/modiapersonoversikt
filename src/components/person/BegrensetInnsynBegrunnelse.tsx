import * as React from 'react';
import { BegrensetInnsynTyper } from '../../models/person/person';

interface Props {
    begrunnelseType: BegrensetInnsynTyper;
}

function BegrensetInnsynBegrunnelse({begrunnelseType}: Props) {
    switch (begrunnelseType) {
        case BegrensetInnsynTyper.Kode6:
            return <>Bruker har kode 6 adresse</>;
        case BegrensetInnsynTyper.Kode7:
            return <>Bruker har kode 7 adresse</>;
        case BegrensetInnsynTyper.EgenAnsatt:
            return <>Bruker faller inn under begrensning for egen ansatt</>;
        default:
            return <>Ukjent sikkerhetsbegrensing</>;
    }
}

export default BegrensetInnsynBegrunnelse;