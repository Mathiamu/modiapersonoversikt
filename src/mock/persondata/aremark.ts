import {
    AdresseBeskyttelse,
    EgenAnsatt,
    ForelderBarnRelasjon,
    ForelderBarnRelasjonRolle,
    FullmaktsRolle,
    Kjonn,
    LocalDate,
    LocalDateTime,
    Person,
    PersonStatus,
    SikkerhetstiltakType,
    SivilstandType,
    Skifteform
} from '../../app/personside/visittkort-v2/PersondataDomain';

export const barnAremark: ForelderBarnRelasjon[] = [
    {
        ident: '12345678910',
        rolle: ForelderBarnRelasjonRolle.BARN,
        navn: [
            {
                fornavn: 'BARN1',
                mellomnavn: null,
                etternavn: 'BARNESEN'
            }
        ],
        fodselsdato: ['2008-03-15' as LocalDate],
        alder: 13,
        adressebeskyttelse: [
            {
                kode: AdresseBeskyttelse.UGRADERT,
                beskrivelse: 'UGRADERT'
            }
        ],
        bostedAdresse: [
            {
                linje1: 'Islandsgate 49',
                linje2: '7693 Hovedøya',
                linje3: 'Norge'
            }
        ],
        personstatus: [
            {
                kode: PersonStatus.BOSATT,
                beskrivelse: 'BOSATT'
            }
        ]
    }
];

export const aremark: Person = {
    fnr: '10108000398',
    navn: [
        {
            fornavn: 'AREMARK',
            mellomnavn: null,
            etternavn: 'TESTFAMILIEN'
        }
    ],
    kjonn: [
        {
            kode: Kjonn.M,
            beskrivelse: 'Mann'
        }
    ],
    fodselsdato: ['1980-10-10' as LocalDate],
    alder: 41,
    dodsdato: [],
    bostedAdresse: [
        {
            linje1: 'Islandsgate 49',
            linje2: '7693 Hovedøya',
            linje3: 'Norge'
        }
    ],
    kontaktAdresse: [
        {
            linje1: 'Aremarkveien 1',
            linje2: '0320 Aremark',
            linje3: null
        }
    ],
    navEnhet: {
        id: '0219',
        navn: 'NAV Bærum'
    },
    statsborgerskap: [
        {
            land: {
                kode: 'NOR',
                beskrivelse: 'NORGE'
            },
            gyldigFraOgMed: '1980-10-10' as LocalDate,
            gyldigTilOgMed: null,
            erHistorisk: false
        }
    ],
    adressebeskyttelse: [
        {
            kode: AdresseBeskyttelse.KODE6,
            beskrivelse: 'Ingen'
        }
    ],
    sikkerhetstiltak: [
        {
            type: SikkerhetstiltakType.TFUS,
            gyldigFraOgMed: '2005-02-13' as LocalDate,
            gyldigTilOgMed: '2030-02-15' as LocalDate
        }
    ],
    erEgenAnsatt: EgenAnsatt.JA,
    personstatus: [
        {
            kode: PersonStatus.BOSATT,
            beskrivelse: 'Bosatt i Norge'
        }
    ],
    sivilstand: [
        {
            type: {
                kode: SivilstandType.GIFT,
                beskrivelse: 'Gift'
            },
            gyldigFraOgMed: '2005-12-12' as LocalDate
        }
    ],
    foreldreansvar: [
        {
            ansvar: 'felles',
            ansvarlig: {
                fornavn: 'Test',
                etternavn: 'Testesen',
                mellomnavn: null
            },
            ansvarsubject: {
                fornavn: 'Barn',
                etternavn: 'Barnesen',
                mellomnavn: null
            }
        }
    ],
    deltBosted: [
        {
            startdatoForKontrakt: '2000-10-10' as LocalDate,
            sluttdatoForKontrakt: '2005-05-07' as LocalDate,
            adresse: {
                linje1: `Adresseveien 1`,
                linje2: '0000 Aremark',
                linje3: null
            }
        }
    ],
    dodsbo: [
        {
            adressat: {
                advokatSomAdressat: {
                    kontaktperson: {
                        fornavn: 'Advokat',
                        mellomnavn: null,
                        etternavn: 'Navnesen'
                    },
                    organisasjonsnavn: 'Advokatkontoret Aremark',
                    organisasjonsnummer: '01234567891011'
                },
                organisasjonSomAdressat: null,
                personSomAdressat: null
            },
            adresse: {
                linje1: 'Elgelia 20, 0000 Aremark',
                linje2: null,
                linje3: null
            },
            registrert: '2010-02-02' as LocalDate,
            skifteform: Skifteform.OFFENTLIG
        }
    ],
    fullmakt: [
        {
            motpartsPersonident: '123456789',
            motpartsPersonNavn: {
                fornavn: 'Navn',
                mellomnavn: null,
                etternavn: 'Navnesen'
            },
            motpartsRolle: FullmaktsRolle.FULLMEKTIG,
            omraade: ['*'],
            gyldigFraOgMed: '2015-01-01' as LocalDate,
            gyldigTilOgMed: '2017-12-12' as LocalDate
        }
    ],
    vergemal: [
        {
            ident: '21042900076',
            navn: {
                fornavn: 'Simen',
                mellomnavn: null,
                etternavn: 'Solli'
            },
            vergesakstype: 'voksen',
            omfang: 'personligeOgOekonomiskeInteresser',
            embete: 'Fylkesmannen i Troms og Finnmark',
            gyldighetstidspunkt: '2016-03-27' as LocalDate,
            opphoerstidspunkt: '2017-06-30' as LocalDate
        }
    ],
    tilrettelagtKommunikasjon: {
        talesprak: [
            {
                kode: 'NO',
                beskrivelse: 'Norsk'
            }
        ],
        tegnsprak: [
            {
                kode: 'NO',
                beskrivelse: 'Norsk'
            }
        ]
    },
    telefonnummer: [
        {
            retningsnummer: {
                kode: '+47',
                beskrivelse: 'retningsnummer er norsk'
            },
            identifikator: '99009900',
            sistEndret: '2017-10-10T10:15:30' as LocalDateTime,
            sistEndretAv: 'TESTFAMILIEN AREMARK',
            prioritet: 1
        }
    ],
    kontaktOgReservasjon: {
        personident: '10108000398',
        reservasjon: 'true',
        epostadresse: {
            value: 'epost@nav.no',
            sistOppdatert: null,
            sistVerifisert: '2013-01-01' as LocalDate
        },
        mobiltelefonnummer: {
            value: '90000000',
            sistOppdatert: null,
            sistVerifisert: null
        }
    },
    bankkonto: {
        kontonummer: '1234567890',
        banknavn: 'DNB ASA',
        sistEndret: '2006-03-15T00:00:00' as LocalDateTime,
        sistEndretAv: '1010800 BD03',

        bankkode: null,
        swift: 'DNBANOKKXXX',
        landkode: null,
        adresse: {
            linje1: 'Bankveien 1,',
            linje2: '0357 Bankestad',
            linje3: null
        },
        valuta: {
            kode: 'NOK',
            beskrivelse: 'Norske kroner'
        }
    },
    forelderBarnRelasjon: barnAremark
};
