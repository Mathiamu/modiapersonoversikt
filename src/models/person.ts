import { Kodeverk } from './kodeverk';

export interface Person {
    navn: Navn;
    kjønn: Kjønn;
    geografiskTilknytning?: string;
    fødselsnummer: string;
    alder: number;
    diskresjonskode?: string;
    bankkonto?: Bankkonto;
    statsborgerskap?: string;
    personstatus: Bostatus;
    sivilstand: Kodeverk;
    familierelasjoner: Familierelasjon[];
}

export interface Navn {
    sammensatt: string;
    fornavn: string;
    mellomnavn: string;
    etternavn: string;
}

export interface Bankkonto {
    erNorskKonto: boolean;
    kontonummer: number;
    bank: string;
    sistEndret: string;
    sistEndretAv: string;
}

export interface Familierelasjon {
    harSammeBosted: boolean;
    rolle: Relasjonstype;
    tilPerson: {
        navn: Navn;
        alder: number;
        fødselsnummer: string;
        personstatus: Bostatus;
        kjonn: Kjønn
    };
}

export interface Bostatus {
    dødsdato?: string;
    bostatus?: string;
}

export enum BostatusTyper {
    Død = 'DØD',
    Utvandret = 'UTVA'
}

export enum Relasjonstype {
    Barn = 'BARN',
    Samboer = 'SAMBOER',
    Partner = 'PARTNER',
    Ektefelle = 'EKTE',
    Gift = 'GIFT'
}

export enum Sivilstand {
    Gift = 'GIFT',
    Ugift = 'UGIFT'
}

export enum Kjønn {
    Mann = 'M',
    Kvinne = 'K'
}

export function erDød(personstatus: Bostatus) {
    return personstatus.dødsdato || personstatus.bostatus === BostatusTyper.Død;
}

export function getBarnUnder21(familierelasjoner: Familierelasjon[]) {
    return familierelasjoner
        .filter(relasjon => relasjon.rolle === Relasjonstype.Barn)
        .filter(relasjon => relasjon.tilPerson.alder <= 21);
}

export function getPartner(person: Person) {
    const aktuelleRelasjoner = [
        Relasjonstype.Ektefelle,
        Relasjonstype.Gift,
        Relasjonstype.Partner,
        Relasjonstype.Samboer
    ];
    return person.familierelasjoner.find(relasjon => aktuelleRelasjoner.includes(relasjon.rolle));
}
