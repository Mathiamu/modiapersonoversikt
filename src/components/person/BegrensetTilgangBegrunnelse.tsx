import * as React from 'react';
import { BegrensetTilgangTyper } from '../../models/person/person';

interface Props {
    begrunnelseType: BegrensetTilgangTyper;
}

function BegrensetTilgangBegrunnelse({begrunnelseType}: Props) {
    switch (begrunnelseType) {
        case BegrensetTilgangTyper.Kode6:
            return <>Bruker har strengt fortrolig adresse</>;
        case BegrensetTilgangTyper.Kode7:
            return <>Bruker har fortrolig adresse</>;
        case BegrensetTilgangTyper.EgenAnsatt:
            return <>Bruker faller inn under begrensning for egen ansatt</>;
        case BegrensetTilgangTyper.DefaultFraBackEnd:
            return <>Ukjent sikkerhetsbegrensing</>;
        default:
            return <>Feil sikkerhetsbegrensing</>;
    }
}

export default BegrensetTilgangBegrunnelse;