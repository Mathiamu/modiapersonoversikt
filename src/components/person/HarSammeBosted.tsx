import * as React from 'react';
import { erDod, Familierelasjon } from '../../models/person/person';

interface Props {
    relasjon?: Familierelasjon;
}

function BorMedBruker({ relasjon }: Props) {
    const personErDod = relasjon && erDod(relasjon.tilPerson.personstatus);
    if (relasjon?.harSammeBosted === undefined || personErDod) {
        return null;
    } else if (relasjon.harSammeBosted) {
        return <>Bor med bruker</>;
    } else {
        return <>Bor ikke med bruker</>;
    }
}

export default BorMedBruker;
