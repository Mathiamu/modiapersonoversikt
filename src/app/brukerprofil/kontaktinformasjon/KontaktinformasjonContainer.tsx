import * as React from 'react';
import styled from 'styled-components';
import { Undertittel } from 'nav-frontend-typografi';
import { Person } from '../../../models/person/person';
import { KodeverkResponse } from '../../../models/kodeverk';
import KontaktinformasjonForm from './KontaktinformasjonForm';
import RestResourceConsumer from '../../../rest/consumer/RestResourceConsumer';

interface Props {
    person: Person;
}

const NavKontaktinformasjonWrapper = styled.div`
    margin-top: 2em;
`;

function KontaktinformasjonFormContainer(props: Props) {
    return (
        <div>
            <Undertittel>Kontaktinformasjon</Undertittel>
            <RestResourceConsumer<KodeverkResponse> getResource={restResources => restResources.retningsnummer}>
                {retningsnummer => (
                    <NavKontaktinformasjonWrapper>
                        <KontaktinformasjonForm person={props.person} retningsnummerKodeverk={retningsnummer} />
                    </NavKontaktinformasjonWrapper>
                )}
            </RestResourceConsumer>
        </div>
    );
}

export default KontaktinformasjonFormContainer;
