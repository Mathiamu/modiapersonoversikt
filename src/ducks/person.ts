import { Person } from '../models/person';
import { Action, Dispatch } from 'redux';
import { FetchError, FetchSuccess, RestActions, STATUS } from './utils';

export interface PersonState {
    data: Person;
    status: STATUS;
    error: string | null;
}

const mockPerson = {
    fornavn: 'Aremark',
    etternavn: 'Testfamilien',
    fodselsnummer: '10108000398'
};

const reducerName = 'person/';

const actionNames = {
    PENDING: reducerName + STATUS.PENDING,
    OK: reducerName + STATUS.OK,
    ERROR: reducerName + STATUS.ERROR
};

const initialState: PersonState = {
    data: mockPerson,
    status: STATUS.NOT_STARTED,
    error: null
};

function mockFetch(fodselsnummer: string) {
    return new Promise((resolve, reject) => {
        setTimeout(() => { resolve(mockPerson); }, 2500);
    });
}

export function hentPerson(fodselsnummer: string) {
    return (dispatch: Dispatch<Action>) => {
        dispatch({ type: actionNames.PENDING});
        return mockFetch(fodselsnummer)
            .then((person) => {
                dispatch({type: actionNames.OK, data: person});
                return person;
            })
            .catch((err) => {
                dispatch({type: actionNames.ERROR, error: err});
            });
    };
}

export default function reducer(state: PersonState = initialState, action: RestActions<Person>): PersonState {
    switch (action.type) {
        case actionNames.PENDING:
            return {...state, status: STATUS.PENDING };
        case actionNames.OK:
            return {...state, status: STATUS.OK, data: (<FetchSuccess<Person>> action).data};
        case actionNames.ERROR:
            return {...state, status: STATUS.ERROR, error: (<FetchError> action).error};
        default:
            return state;
    }
}