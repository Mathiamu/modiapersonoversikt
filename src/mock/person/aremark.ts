import { Diskresjonskoder } from '../../constants';
import { Person } from '../../models/person';

export const aremark: Person = {
    fødselsnummer: '10108000398',
    kjønn: 'M',
    geografiskTilknytning: '0118',
    alder: 42,
    navn: {
        sammensatt: 'AREMARK TESTFAMILIEN',
        fornavn: 'AREMARK',
        mellomnavn: '',
        etternavn: 'TESTFAMILIEN',
    },
    diskresjonskode: Diskresjonskoder.FORTROLIG_ADRESSE,
    statsborgerskap: 'NORSK',
    status: {
        dødsdato: undefined,
        bostatus: undefined
    },
    sivilstand: {
        value: 'GIFT',
        beskrivelse: 'Gift'
    },
    familierelasjoner: []
};