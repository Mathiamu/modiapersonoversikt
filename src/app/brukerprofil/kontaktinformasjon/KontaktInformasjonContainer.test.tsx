import * as React from 'react';
import * as renderer from 'react-test-renderer';
import TestProvider from '../../../test/Testprovider';
import { aremark } from '../../../mock/person/aremark';
import KontaktinformasjonContainer from './KontaktinformasjonContainer';

test('Viser form for endre kontaktinformasjon', () => {

    const endreTilrettelagtKommunikasjonForm = renderer.create(
        <TestProvider>
            <KontaktinformasjonContainer person={aremark} />
        </TestProvider>
    );

    expect(endreTilrettelagtKommunikasjonForm.toJSON()).toMatchSnapshot();
});
