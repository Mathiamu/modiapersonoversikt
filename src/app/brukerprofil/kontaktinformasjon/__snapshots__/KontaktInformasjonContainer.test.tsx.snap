// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Viser form for endre kontaktinformasjon 1`] = `
<div>
  <h2
    className="typo-undertittel"
  >
    Kontaktinformasjon
  </h2>
  <div
    className="sc-kpOJdX cTiSZX"
  >
    <form
      onSubmit={[Function]}
    >
      <p
        className="typo-undertekst-bold"
      >
        Telefonnummer bruker ønsker å bli oppringt på
      </p>
      <div
        className="sc-kgoBCf bepbwp"
      >
        <div
          className="sc-kGXeez iWjhrf"
        >
          <p
            className="typo-ingress"
          >
            Mobiltelefon
          </p>
          <div
            className="sc-ifAKCX WOpjk"
          >
            <div
              className="sc-htpNat dUHUAp"
            >
              <div
                className="skjemaelement"
              >
                <label
                  className="skjemaelement__label"
                  htmlFor="Retningsnummer Mobiltelefon"
                >
                  Landkode
                </label>
                <div
                  className="selectContainer input--m"
                >
                  <select
                    className="skjemaelement__input"
                    disabled={false}
                    id="Retningsnummer Mobiltelefon"
                    onChange={[Function]}
                    value="+47"
                  >
                    <option
                      disabled={true}
                      value=""
                    >
                      Velg retningsnummer
                    </option>
                    <option
                      value="+47"
                    >
                      Norge
                       (
                      +47
                      )
                    </option>
                    <option
                      value="+54"
                    >
                      Argentina
                       (
                      +54
                      )
                    </option>
                    <option
                      value="+44"
                    >
                      Storbritannia (UK)
                       (
                      +44
                      )
                    </option>
                    <option
                      value="+46"
                    >
                      Sverige
                       (
                      +46
                      )
                    </option>
                  </select>
                </div>
                <div
                  aria-live="assertive"
                  role="alert"
                />
              </div>
            </div>
            <div
              className="sc-bxivhb eILgmy"
            >
              <div
                className="skjemaelement"
              >
                <label
                  className="skjemaelement__label"
                  htmlFor="Mobiltelefon"
                >
                  Telefonnummer
                </label>
                <input
                  className="skjemaelement__input input--xxl"
                  id="Mobiltelefon"
                  name={undefined}
                  onChange={[Function]}
                  type="text"
                  value="998 87 766"
                />
                <div
                  aria-live="assertive"
                  role="alert"
                />
              </div>
            </div>
          </div>
          <span
            className="sc-bwzfXH opmLB"
          >
            <p
              className="typo-etikett-liten"
            >
              Endret 
              21.06.2014
               
              BRUKER
            </p>
          </span>
        </div>
        <div
          className="sc-kGXeez iWjhrf"
        >
          <p
            className="typo-ingress"
          >
            Hjemmenummer
          </p>
          <div
            className="sc-ifAKCX WOpjk"
          >
            <div
              className="sc-htpNat dUHUAp"
            >
              <div
                className="skjemaelement"
              >
                <label
                  className="skjemaelement__label"
                  htmlFor="Retningsnummer Hjemmenummer"
                >
                  Landkode
                </label>
                <div
                  className="selectContainer input--m"
                >
                  <select
                    className="skjemaelement__input"
                    disabled={false}
                    id="Retningsnummer Hjemmenummer"
                    onChange={[Function]}
                    value=""
                  >
                    <option
                      disabled={true}
                      value=""
                    >
                      Velg retningsnummer
                    </option>
                    <option
                      value="+47"
                    >
                      Norge
                       (
                      +47
                      )
                    </option>
                    <option
                      value="+54"
                    >
                      Argentina
                       (
                      +54
                      )
                    </option>
                    <option
                      value="+44"
                    >
                      Storbritannia (UK)
                       (
                      +44
                      )
                    </option>
                    <option
                      value="+46"
                    >
                      Sverige
                       (
                      +46
                      )
                    </option>
                  </select>
                </div>
                <div
                  aria-live="assertive"
                  role="alert"
                />
              </div>
            </div>
            <div
              className="sc-bxivhb eILgmy"
            >
              <div
                className="skjemaelement"
              >
                <label
                  className="skjemaelement__label"
                  htmlFor="Hjemmenummer"
                >
                  Telefonnummer
                </label>
                <input
                  className="skjemaelement__input input--xxl"
                  id="Hjemmenummer"
                  name={undefined}
                  onChange={[Function]}
                  type="text"
                  value=""
                />
                <div
                  aria-live="assertive"
                  role="alert"
                />
              </div>
            </div>
          </div>
        </div>
        <p
          className="typo-ingress"
        >
          Jobbnummer
        </p>
        <div
          className="sc-ifAKCX WOpjk"
        >
          <div
            className="sc-htpNat dUHUAp"
          >
            <div
              className="skjemaelement"
            >
              <label
                className="skjemaelement__label"
                htmlFor="Retningsnummer Jobbnummer"
              >
                Landkode
              </label>
              <div
                className="selectContainer input--m"
              >
                <select
                  className="skjemaelement__input"
                  disabled={false}
                  id="Retningsnummer Jobbnummer"
                  onChange={[Function]}
                  value=""
                >
                  <option
                    disabled={true}
                    value=""
                  >
                    Velg retningsnummer
                  </option>
                  <option
                    value="+47"
                  >
                    Norge
                     (
                    +47
                    )
                  </option>
                  <option
                    value="+54"
                  >
                    Argentina
                     (
                    +54
                    )
                  </option>
                  <option
                    value="+44"
                  >
                    Storbritannia (UK)
                     (
                    +44
                    )
                  </option>
                  <option
                    value="+46"
                  >
                    Sverige
                     (
                    +46
                    )
                  </option>
                </select>
              </div>
              <div
                aria-live="assertive"
                role="alert"
              />
            </div>
          </div>
          <div
            className="sc-bxivhb eILgmy"
          >
            <div
              className="skjemaelement"
            >
              <label
                className="skjemaelement__label"
                htmlFor="Jobbnummer"
              >
                Telefonnummer
              </label>
              <input
                className="skjemaelement__input input--xxl"
                id="Jobbnummer"
                name={undefined}
                onChange={[Function]}
                type="text"
                value=""
              />
              <div
                aria-live="assertive"
                role="alert"
              />
            </div>
          </div>
        </div>
      </div>
      <div
        className="sc-chPdSV gopUKn"
      >
        <button
          className="knapp knapp--standard knapp--disabled"
          disabled={true}
          onClick={[Function]}
          type="submit"
        >
          Avbryt
        </button>
        <button
          className="knapp knapp--hoved knapp--disabled"
          disabled={true}
          type="submit"
        >
          Endre telefonnummer
        </button>
      </div>
    </form>
  </div>
</div>
`;
